### PR TITLE
Fix/display of current network switch updates

### PR DESCRIPTION
**Pull request by georgewrmarshall**
## Explanation

Some icon and color updates to ensure consistency and theme compatibility for the confirmation-network-switch and permsions-redirect components

- Also catching some other unused colors and updating

## Screenshots and Screencasts

https://user-images.githubusercontent.com/8112138/160875402-a69eb87e-af30-4dbd-9b49-9aa89e9926d2.mov

https://user-images.githubusercontent.com/8112138/160875408-db89f779-a65a-4221-a864-ada9294edb6f.mov

Screen Shot 2022-03-30 at 8 54 02 AM
Screen Shot 2022-03-30 at 8 55 21 AM
Screen Shot 2022-03-30 at 8 54 11 AM
Screen Shot 2022-03-30 at 8 54 44 AM

Screen Shot 2022-03-30 at 8 05 10 AM
Screen Shot 2022-03-30 at 8 06 01 AM
Screen Shot 2022-03-30 at 8 05 32 AM
Screen Shot 2022-03-30 at 8 05 46 AM

### CODE REVIEW

File: ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.stories.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    newNetwork: {
+      chainId: 'chainId',
+      name: 'name',
+    },
+  },
+};
+
+export const DefaultStory = (args) => <ConfirmationNetworkSwitch {...args} />;
+
+DefaultStory.storyName = 'Default';
```
Comment (georgewrmarshall) [inline ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.stories.js:22]: We should try to add stories for any new components.

File: ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ export default function ConfirmationNetworkSwitch({ newNetwork }) {
             size={64}
           />
           <Typography
-            color={COLORS.BLACK}
+            color={COLORS.TEXT_DEFAULT}
             variant={TYPOGRAPHY.H6}
             fontWeight={FONT_WEIGHT.NORMAL}
             align={TEXT_ALIGN.CENTER}
```
Comment (georgewrmarshall) [inline ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js:99]: Use font-awesome icons in favor of svgs to ensure we have the correct licensing and allows us to access the css to apply the theme compatible colors via css

File: ui/pages/confirmation/confirmation.js [hunk 3]
```diff
@@ -210,7 +210,7 @@ export default function ConfirmationPage() {
           />
           <Chip
             label={stripHttpsScheme(originMetadata.origin)}
-            borderColor={COLORS.WHITE}
+            borderColor={COLORS.BORDER_DEFAULT}
           />
         </Box>
         <MetaMaskTemplateRenderer sections={templatedValues.content} />
```
Comment (georgewrmarshall) [inline ui/pages/confirmation/confirmation.js:216]: Udpating color that no longer exists.

File: ui/pages/confirmation/templates/switch-ethereum-chain.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ function getValues(pendingApproval, t, actions) {
         children: t('switchEthereumChainConfirmationDescription'),
         props: {
           variant: TYPOGRAPHY.H7,
-          color: COLORS.UI4,
+          color: COLORS.TEXT_ALTERNATIVE,
           align: 'center',
           boxProps: {
             padding: [0, 4, 0, 4],
```
Comment (georgewrmarshall) [inline ui/pages/confirmation/templates/switch-ethereum-chain.js:53]: Updating color that no longer exists to theme compatible color

File: ui/pages/permissions-connect/redirect/permissions-redirect.component.js [hunk 5]
```diff
@@ -21,9 +21,8 @@ export default function PermissionsRedirect({ subjectMetadata }) {
...
             className="permissions-redirect__site-icon"
           />
-          <div className="permissions-redirect__center-icon">
-            <span className="permissions-redirect__check" />
-            <img src="./images/broken-line.svg" alt="broken-line" />
+          <div className="permissions-redirect__line">
+            <i className="fa fa-check-circle fa-2x permissions-redirect__check" />
           </div>
           <SiteIcon
             icon="/images/logo/metamask-fox.svg"
```
Comment (georgewrmarshall) [inline ui/pages/permissions-connect/redirect/permissions-redirect.component.js:28]: Making the permissions redirect consistent with the confirmation network switch component as they are very similar.

**Comment by brad-decker** [comment]
Looks good to me, we'll just wait for the base to merge to develop then rebase and merge this one.

**Comment by darkwing** [comment]
We'll need to get this wrapped into the next release (v10.13.0).

**Comment by georgewrmarshall** [comment]
Hey @amerkadicE, that was my bad I didn't make it prominent enough. I've rebased from your PR and updated the description with new screenshots and screencasts. Let me know if you're ok to merge.

**Review by amerkadicE — APPROVED** [review]
LGTM

**Pull request merged**